index.html

```diff
@@ -0,0 +1,708 @@
+                    <h2>1,482 smashing members</h2>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Beatae reiciendis repellendus neque
+                        dolorum
+                        dolorem ut.</p>
+                    <button class="explore explore-btn btn">
+
+                        Lorem ipsum, dolor
+
+                    </button>
+                </div>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="load_more">
+                <div class="col-12">
+                    <a class="load_more_link" href="#">Lorem ipsum dolot
+                    </a>
+                </div>
+            </div>
+        </div>
+
+        <section class="row banner-second">
+            <div class="col-medium-12">
+                <h3>
+                    Smashing Print #1: Ethics & Privacy
+                </h3>
+                <p>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor, repellendus fugiat ex autem adipisci
+                    veritatis iusto vero id placeat quam reprehenderit, et quisquam explicabo assumenda nostrum, sed
+                    earum
+                    maxime nesciunt?
+                </p>
+                <button class="explore btn explore-btn">
+
+                    Lorem ipsum dolor
+
+                </button>
+
+            </div>
+        </section>
+        <div class="row">
+            <div class="col-medium-12 banner">
+                <img src="https://d33wubrfki0l68.cloudfront.net/148069bc1849f0c7b2304c945073dbdfdb6341f0/b12bb/images/printed-mag/printmag-homepage-opt.svg"
+                    alt="printed-image">
+            </div>
+        </div>
+
+        <div class="person-of-the-week">
+            <article class="row">
+                <div class="col-large-3 col-medium-6 first_person">
+                    <img src="../Smashing-Magazine/assets/img/Kelly.png" alt="Kelly photo">
+
+                </div>
+                <div class="col-large-9 col-medium-6 second_person">
+                    <h3>Person of the Week
+
+                    </h3>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Dolores atque necessitatibus
+                        dignissimos id
+                        adipisci molestiae ipsam ea laborum
+                        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Dolores atque necessitatibus
+                        dignissimos
+                        id adipisci molestiae ipsam ea laborum l
+
+                    </p>
+                </div>
+            </article>
+            <div class="row person">
+                <article class="col-large-6 col-medium-12">
+                    <div class="row">
+                        <div class="col-medium-2 persons_photo">
+                            <img src="../Smashing-Magazine/assets/img/rachel-andrew-tw-profile.jpg" alt="Persons photo">
```

Comment:
You still have to correct the route to the images so it shows when someone clones your project.